src/testdrive/src/action.rs

```diff
@@ -93,9 +93,12 @@ pub struct Config {
     pub backoff_factor: f64,
 
     // === Materialize options. ===
-    /// The connection parameters for the materialized instance that testdrive
+    /// The pgwire connection parameters for the materialized instance that testdrive
     /// will connect to.
     pub materialized_pgconfig: tokio_postgres::Config,
+    /// The port for the public endpoints of the materialized instance that
+    /// testdrive will connect to via HTTP.
+    pub materialized_http_port: u16,
```

Comment:
do you need the port and the http_addr below? or can the addr just include the port?

Reply:
Are you asking about replacing the lines below (155-156)
```
    materialized_sql_addr: String,
    materialized_http_addr: String,
```
with
```
    materialized_host: String,
    materialized_sql_port: String,
    materialized_http_port: String
```
?

We could do the replacement if you think it looks neater. It's several more characters of refactoring across all testdrive tests.